src/main.rs

```diff
@@ -92,30 +109,14 @@ fn do_update_counting(arg: String) -> i32 {
 
     let fs = BufReader::new(reader);
     match kind {
-        CommandKind::Apt => {
-            let num = fs.lines().skip(2).count().to_string();
-            num.parse::<i32>().unwrap()
-        }
-        CommandKind::Portage => {
-            let num = fs.lines().count().to_string();
-            if num.trim_end_matches('\n') != "matches" {
-                num.trim_end_matches('\n').parse::<i32>().unwrap_or(1)
-            } else {
-                0
-            }
```

Comment:
this block of code was turning a number into a string, and then checking if that string had "matches" in it, which of course it wouldn't, it's a number